backend/fms_report/services/report_data_preparation.py

```diff
@@ -51,8 +44,7 @@ def prepare_production_report_data(log):
     queryset = queryset.select_related("dataset__experiment_run__id")
     queryset = queryset.select_related("dataset__experiment_run__instrument__type__type")
     queryset = queryset.select_related("dataset__experiment_run__container__kind")
-    queryset = queryset.select_related("dataset__project__external_id")
-    queryset = queryset.select_related("dataset__project__name")
+    queryset = queryset.select_related("dataset__project_id")
```

Comment:
I wonder if you can combine all those selected fields into one `select_related` call like: `queryset = queryset.select_related("dataset__lane", "dataset__experiment_run__start_date", ...)`

Reply:
i find it more readable that way ...